app/views/pets/results.html.erb

```diff
@@ -1,70 +1,73 @@
-<div class="panel panel-info">
-	<div class="panel-heading">
-		<h3 class="panel-title">Sign up for PetAlerts for this Search</h3>
-	</div>
-	<div class="panel-body">
-		<p>
-			You can receive alerts by email or text message when a new pet is added that matches your search conditions. We will not share your contact information or spam you. You can cancel an alert any time by <%= link_to 'clicking here', unsubscribe_url %>.
+<div class= "container">
+	<div class= "subscription-form">
+	<h2 class="call-to-action centered">Get Alerts for this Search
+	</h2>
+		<p class= "centered">
+			Receive alerts by email or text message when a new pet matching this search is posted.
 		</p>
 
 		<%= form_for @subscription, url: subscribe_url, html: { method: 'post' } do |f| %>
-		<div class='form-group col-xs-4'>
-			<%= f.label 'species' %>
-			<%= f.text_field 'species', class: 'form-control', readonly: true %>
+
+		<div class='subscription-readonly-input'>
+			<%= f.label 'species:' %>
```

Comment:
@Tallys The colon in these labels means they can't be clicked to activate the form element. I think this should look like `f.label 'species', 'species:'` if you want the colon. Same goes for the rest of the form elements.